### ISSUE TITLE
E-mail field does not appear in mail notification

### ISSUE THREAD

**Issue by jessedobbelaere** [verbb/formie#2198]
### Describe the bug

I have a form where two e-mail notifications are sent on submission with Formie:
1. A confirmation mail to the external person submitting the form
2. An e-mail to the admin of the website

![2024-12-21 at 00 48 41@2x](https://github.com/user-attachments/assets/7b826ec5-8ccc-444c-a15a-817aa71f388a)

The form contains a standard e-mail address field with handle `emailAddressField` and label `E-mailadres`.
For both e-mail notifications, in the "Email Content" rich text builder, i have inserted the "E-mailadres" field. 

**I noticed that in the customer confirmation mail the e-mail address is visible. However in the admin e-mail, the e-mailaddress is blank.** 

Even weirder, when I print **Visible Fields** in the e-mail notification, it does show the field value. Any idea why that would happen?

![2024-12-21 at 00 41 14@2x](https://github.com/user-attachments/assets/ebca64b4-c08f-447d-b198-2222cd7bf13e)

(Another bug is that I have a Hidden subfield "Uw adres opzoeken" inside an Address field, but it does show up when I print the Visible Fields 😅 but let's ignore that one for now)

### Steps to reproduce

1. Create a field with an e-mailaddress field with a custom handle `emailAddressField` and custom label
2. Create two e-mail notifications, one for the site admin and one for the form user (notification Recipients is set to the Email address field!)
4. Add the E-mail Address field label in the email notification "Email Content" editor
5. Submit the form and grab both e-mails
6. I see one notification where the e-mailaddress shows up and one where it doesn't 🤔 

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: Yes

### Craft CMS version

5.5.7

### Plugin version

3.0.15

### Multi-site?

Yes

**Comment by engram-design** [comment]
I can't seem to replicate that I'm afraid. Can you send through a support request via Formie > Settings > Get Support? That way I can at least replicate your form and notifications exactly as you have them.

**Comment by jessedobbelaere** [comment]
I did some debugging and I think the `formie_notifications.content` table column contains `{field:emailField}` while the other notification has `{field:emailAddressField}`.

```
{
    "type": "variableTag",
    "attrs": {
        "label": "E-mailadres",
        "value": "{field:emailField}"    <---- should be field:emailAddressField! 
    }
},
```

So i think that I have created the notification content message, then – while still tweaking the form – I might have renamed the field handle from `emailField` → `emailAddressField`. The Notification content editor only shows the variable by label, so you don't notice that the underlying field handle is wrong. When you modify the notification content editor, it does not resave or invalidate the current variables. I have to remove the variable and insert it again. Now it works 🎉 

![2024-12-21 at 20 09 09@2x](https://github.com/user-attachments/assets/c9954c6a-ca99-4b61-b5c4-1d20e14c2911)
